client/extensions/woocommerce/app/products/product-variation-types-form.js

```diff
@@ -97,19 +97,19 @@ export default class ProductVariationTypesForm extends Component {
 
 		return (
 			<div className="products__variation-types-form-wrapper">
-				<strong>{ i18n.translate( 'Variation types' ) }</strong>
+				<strong>{ i18n.translate( 'Okay, let\'s add some variations!' ) }</strong>
 				<p>
 					{ i18n.translate(
-						'Let\'s add some variations! A common {{em}}variation type{{/em}} is color. ' +
-						'The {{em}}values{{/em}} would be the colors the product is available in.',
+						'A common variation type is color. ' +
+						'The values would be the colors the product is available in.',
 						{ components: { em: <em /> } }
 					) }
 				</p>
 
 				<div className="products__variation-types-form-group">
 					<div className="products__variation-types-form-labels">
 						<FormLabel className="products__variation-types-form-label">{ i18n.translate( 'Variation type' ) }</FormLabel>
-						<FormLabel>{ i18n.translate( 'Variation values' ) }</FormLabel>
+						<FormLabel>{ i18n.translate( 'Values' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **90** times:
`translate( 'Value' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).